test/workload.yaml

```diff
@@ -10,8 +10,12 @@
   spec:
     interface: eth0
     profiles: [prof1, prof2]
-    ipNetworks: [1.2.3.4/32, "00:bb::aa/10"]
+    ipNetworks: ["1.2.3.4/32", "00:bb::aa/128"]
+    ipNATs:
+    - externalIP: 10.20.30.40
```

Comment:
Is this how we want to expose these?

What about `workloadIP` and `natIP` or something to make it more clear which IP address is which? Internal / external don't really make sense.